app/views/galleries/_single.haml

```diff
@@ -1,14 +1,17 @@
+-# locals: ( gallery:, icons: nil, character_gallery: nil, is_owner: false, skip_forms: false, hide_minmax: false, klass: '' )
```

Comment:
```suggestion
-# locals: ( gallery:, icons: nil, character_gallery: nil, is_owner: false, skip_forms: false, hide_minmax: false, klass: nil )
```
This way we don't change the value we pass to `class:` below, right?

Reply:
Whoops, I thought I caught all of those

Reply:
I don't think it makes a difference in this case actually! Because we only use it there and don't further manipulate it